scripts/k8s/getOffset.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/python
+
+from kazoo.client import KazooClient
+import json
+import sys
+import time
+
+topic = sys.argv[1]
+
+zk = KazooClient(hosts='zookeeper:2181')
+zk.start()
+try:
+  resp = zk.get("/config/topics/%s" % topic)
+except:
+  print "oldest"
+  sys.exit(0)
+finally:
+  zk.stop()
+
+if len(resp) < 1:
+  print "oldest"
+  sys.exit(0)
+
+data = json.loads(resp[0])
+
+retention=int(data['config'].get('retention.ms', 0))
+segmentSize=int(data['config'].get('segment.ms', 1800000))
+
+if retention == 0:
+  print "oldest"
+  sys.exit(0)
+
+# oldest we can safetly consume from in minutes
+oldest = int((retention - segmentSize)/(1000 * 60))
```

Comment:
is it necessary to convert that into `int()`? i think it should be `int` anyway

Reply:
i want an int not a float

Reply:
i don't think python would make this a float
```
>>> segmentSize=int(100)
>>> retention=int(1234512345)
>>> (retention - segmentSize)/(1000 * 60)
20575
>>> type((retention - segmentSize)/(1000 * 60))
<type 'int'>
```